winstone/src/winstone/StaticResourceServlet.java

```diff
@@ -131,17 +130,17 @@ else if (res.isDirectory()) {
         }
 
         // Send a 304 if not modified
-        else if (!isInclude && (cachedResDate != -1)
-                && (cachedResDate < (System.currentTimeMillis() / 1000L * 1000L))
-                && (cachedResDate >= (res.lastModified() / 1000L * 1000L))) {
-            String mimeType = getServletContext().getMimeType(
-                    res.getName().toLowerCase());
-            if (mimeType != null)
-                response.setContentType(mimeType);
-            response.setStatus(HttpServletResponse.SC_NOT_MODIFIED);
-            response.setContentLength(0);
-            response.flushBuffer();
-        }
+//        else if (!isInclude && (cachedResDate != -1)
```

Comment:
As above, could you add a comment here that links to https://github.com/cubing/tnoodle/issues/215?